apps/settings/elements/browsing_privacy.html

```diff
@@ -21,6 +21,14 @@ <h1 data-l10n-id="browsingPrivacy-header"></h1>
             <details data-l10n-id="privateBrowsingDefaultDetails"></details>
           </gaia-switch>
         </li>
+        <li>
+          <gaia-switch name="privacy.trackingprotection.enabled">
+            <label data-l10n-id="trackingProtection"></label>
+            <details data-l10n-id="trackingProtectionDetails"></details>
+            <a data-l10n-id="learn-more"
```

Comment:
Before adding this functionality to the component, I think I would like to have two "customers" of it - and at that time it would be easy to modify. For now can we include any necessary CSS inside of the settings app CSS, and do something like the following:

```
<details>
  <span data-l10n-id="trackingProtectionDetails"></span><br>
  <a data-l10n-id="learn-more" href="https://..."></a>
</details>
```